docs/dev/fundamentals/rollups.md

```diff
@@ -0,0 +1,40 @@
+# Introduction
+
+To better understand rollups we need to dive briefly into Ethereum and Layer 2 solutions.
+
+The enormous demands from users to transact Ethereum put a limit on the network capacity by causing congestion, resulting in slow transactions and increased gas prices. While this has remained so for a while, an improved solution is needed, one that can maintain scalability.
+That is, to increase finality and achieve high transaction rates per second without having to trade off security. That's where layer 2 solutions shine.
+
+Layer 2 solutions are designed as an extension to Ethereum, offer various solutions poised to be the critical scalability solution to the inherent network congestion on Ethereum. Covering all Layer 2 solutions is beyond the scope of this doc.
+We will go through a brief explainer on rollups in this section.
+
+## optimistic rollups versus zk rollups
```

Comment:
```suggestion
## Optimistic rollups versus zk rollups
```

Seems like we capitalize the title everywhere

Reply:
Yes